src/pages/dmt5/_download-app.tsx

```diff
@@ -25,200 +41,339 @@ const query = graphql`
     }
 `
 
-const DownloadLinkWrapper = styled.div`
+const DownloadLinkWrapper = styled.div<LinkType>`
     display: grid;
-    grid-template-columns: repeat(2, 1fr);
+    grid-template-columns: 1fr 2fr;
+    grid-gap: 8px;
+    margin-top: ${(props) => (props.mt ? props.mt : '')};
+    margin-bottom: ${(props) => (props.mb ? props.mb : '')};
+
+    @media ${device.tabletL} {
+        grid-template-columns: 1fr 1fr;
+    }
+`
+const DownloadLinkMobileWrapper = styled.div<LinkType>`
+    display: flex;
     grid-gap: 8px;
+    margin-top: ${(props) => (props.mt ? props.mt : '')};
```

Comment:
same for these as well

Reply:
Hi @habib-deriv , fixed changes requested. Please review🙏